.github/workflows/pull-request.yaml

```diff
@@ -8,12 +8,12 @@ jobs:
   validate_metrics_docs:
     runs-on: ubuntu-latest
     steps:
-      - name: Install Golang 1.20
-        uses: actions/setup-go@v2
-        with:
-          go-version: '1.20'
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Will renovate update such lines in the future, e.g. like in https://github.com/dominikholler/webdavserver/pull/5 ?

Reply:
Yes, it will.